Makefile

```diff
@@ -3,16 +3,16 @@
 all: clean install lint test
 
 build:
-	npx mega-linter-runner
+	npm_config_registry=https://jfrog.elhub.cloud/artifactory/api/npm/elhub-npm/; npx mega-linter-runner
```

Comment:
can we use NPM_REGISTRY at the top of the script and use it to set the npm_config_registry environment variable in each target that needs it. This makes the script more maintainable because we only need to change the registry URL in one place if it ever changes.
for example:
NPM_REGISTRY=https://jfrog.elhub.cloud/artifactory/api/npm/elhub-npm/
all: clean install lint test
build:
	export npm_config_registry=$(NPM_REGISTRY)
	npx mega-linter-runner